### ISSUE TITLE
Update Vietnamese translations

### ISSUE THREAD

**Issue by betty2310** [squidfunk/mkdocs-material#4935]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### Language availability

- [X] I've checked the [list of available languages](https://bit.ly/33vFDD0)

### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "vi",
  "action.edit": "Chỉnh sửa",
  "action.skip": "Bỏ qua",
  "action.view": "Xem mã nguồn của trang ⬅️",
  "announce.dismiss": "Không hiển thị lại ⬅️",
  "blog.archive": "Lưu trữ⬅️",
  "blog.categories": "Mục ⬅️",
  "blog.categories.in": "Trong ⬅️",
  "blog.continue": "Tiếp tục đọc ⬅️",
  "blog.draft": "Bản nháp⬅️",
  "blog.index": "Quay lại  ⬅️",
  "blog.meta": "Metadata ⬅️",
  "blog.references": "Các liên kết liên quan ⬅️",
  "clipboard.copy": "Sao chép vào bộ nhớ tạm",
  "clipboard.copied": "Đã sao chép",
  "consent.accept": "Đồng ý ⬅️",
  "consent.manage": "Cài đặt ⬅️",
  "consent.reject": "Từ chối⬅️",
  "footer": "Chân trang ⬅️",
  "footer.next": "Sau",
  "footer.previous": "Trước",
  "header": "Đầu trang ⬅️",
  "meta.comments": "Bình luận",
  "meta.source": "Mã nguồn",
  "nav": "Thanh điều hướng ⬅️",
  "readtime.one": "1 phút đọc ⬅️",
  "readtime.other": "# phút đọc ⬅️",
  "rss.created": "RSS feed ⬅️",
  "rss.updated": "RSS feed of updated content ⬅️",
  "search": "Tìm kiếm ⬅️",
  "search.placeholder": "Tìm kiếm",
  "search.share": "Chia sẻ ⬅️",
  "search.reset": "Xoá ⬅️",
  "search.result.initializer": "Initializing search ⬅️",
  "search.result.placeholder": "Nhập để bắt đầu tìm kiếm",
  "search.result.none": "Không tìm thấy tài liệu liên quan",
  "search.result.one": "1 tài liệu liên quan",
  "search.result.other": "# tài liệu liên quan",
  "search.result.more.one": "1 từ khác trong trang ⬅️",
  "search.result.more.other": "# từ khác trong trang ⬅️",
  "search.result.term.missing": "Không ⬅️",
  "select.language": "Chọn ngôn ngữ ⬅️",
  "select.version": "Chọn phiên bản ⬅️",
  "source": "Xem mã nguồn",
  "source.file.contributors": "Contributors ⬅️",
  "source.file.date.created": "Tạo",
  "source.file.date.updated": "Cập nhật lần cuối",
  "tabs": "Tabs ⬅️",
  "toc": "Mục lục",
  "top": "Trở lại mục lục ⬅️"
}[key] }}{% endmacro %}
```

**Comment by squidfunk** [comment]
Thanks for providing the translations! Added in 664af42367c696abf0d7f58e3903fb44579cde89.

**Comment by squidfunk** [comment]
Released as part of [9.0.8](https://github.com/squidfunk/mkdocs-material/releases/tag/9.0.8).
